nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const app = express();
+module.exports = app;
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+const mealsNum = meals.map((meal) => meal.id);
+const reviewMealId = reviews.map((review) => review.mealId);
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("asd");
+});
+// meals with review
+app.get("/meals", async (request, response) => {
+  const mealWithReview = meals.map(meal => {
+    const reviewMatchMeal = reviews.filter(review => review.mealId === meal.id)
+    return {
+      ...meal,
+      reviews: reviewMatchMeal
+    }
+  })
+  response.send(mealWithReview);
+});
+// cheap-meals
+app.get("/cheap-meals", async (request, response) => {
+  const CheapMeal = meals.filter(meal => meal.price < 70)
+  const CheapMealWithReview = CheapMeal.map((meal) => {
```

Comment:
Nitpick(variable naming):
Prefer to use camelCase for variable names, use PascalCase for classes!